scaling_performance/optimizing_storage.adoc

```diff
@@ -13,6 +13,81 @@ toc::[]
 
 == Optimizing Storage
 
+== General Storage Guidance
+
+The following is a matrix which details the storage backend types {product-title} can be used with implementation examples and vendors.
+
+=== Available Storage Options
+[format="csv",width="70%",cols="3"]
+[options="header]
+|=======================
+Type,Examples,Vendors
+hostPath,"hostPath pod mounted volumes.  A specific type of Block Storage.  File paths on the node mounted onto running pods.",Others
```

Comment:
I conceptually disagree with putting hostpath at the same level of the other storage type.
as far as storage type goes hostpath is a block storage.
hostpath is a volume type, as a volume type I categorize it as a local storage.
the other relevant volume types in this discussion are
ephemeral storage (empty dir)  which is missing from your document and it's again block storage
and persistent volumes.

I suggest to talk about these concepts in a separate table.

Reply:
How is it block storage? Does OCP modify permissions on host mount storage? If not, then it isn't block.